### PR TITLE
Tabindex

### PR DESCRIPTION
**Pull request by xahgmah**

### CODE REVIEW

File: dist/videojs-transcript.js [hunk 1]
```diff
@@ -246,17 +247,21 @@ var scrollerProto = function(plugin) {
...
 
+      if (plugin.settings.scrollToCenter){
+        centerPosCorrection = Math.round(parent.clientHeight/2 - element.clientHeight/2);
+      }
       // If the top of the line is above the top of the parent view, were scrolling up,
       // so we want to move the top of the element downwards to match the top of the parent.
-      if (relTop < parent.scrollTop) {
-        newPos = element.offsetTop - parent.offsetTop;
+      if (relTop < parent.scrollTop + centerPosCorrection) {
+        newPos = element.offsetTop - parent.offsetTop -centerPosCorrection;
```
Comment (z4y4ts) [inline dist/videojs-transcript.js]: `- centerPosCorrection`

File: dist/videojs-transcript.js [hunk 2]
```diff
@@ -246,17 +247,21 @@ var scrollerProto = function(plugin) {
       var elementOffsetBottom = element.offsetTop + element.clientHeight;
       var relTop = element.offsetTop - parent.offsetTop;
       var relBottom = (element.offsetTop + element.clientHeight) - parent.offsetTop;
+      var centerPosCorrection;
       var newPos;
 
+      if (plugin.settings.scrollToCenter){
+        centerPosCorrection = Math.round(parent.clientHeight/2 - element.clientHeight/2);
+      }
```
Comment (z4y4ts) [inline dist/videojs-transcript.js:255]: So what about the case `plugin.settings.scrollToCenter == false`?
What's `centerPosCorrection` is going to be?

File: dist/videojs-transcript.js [hunk 3]
```diff
@@ -1,5 +1,5 @@
-/*! videojs-transcript - v0.8.0 - 2016-02-21
-* Copyright (c) 2016 Matthew Walsh; Licensed MIT */
+/*! videojs-transcript - v0.8.0 - 2017-03-20
+* Copyright (c) 2017 Matthew Walsh; Licensed MIT */
```
Comment (z4y4ts) [inline dist/videojs-transcript.js]: I think it's better to leave this header as is and let Matthew worry about it.

**Review by z4y4ts — CHANGES_REQUESTED** [review]

File: dist/videojs-transcript.js (hunk 2 above)
Comment (xahgmah) [reply dist/videojs-transcript.js:255]: On `scrollToCenter == false`  `centerPosCorrection` does nothing.
Set  `centerPosCorrection == 0` by default

**Review by z4y4ts — APPROVED** [review]

**Pull request merged**